test/xbgpu/conftest.py

```diff
@@ -21,22 +21,22 @@
 
 
 @pytest.fixture
-def n_src_streams() -> int:  # noqa: D401
+def n_recv_streams() -> int:  # noqa: D401
     """Number of source streams for an xbgpu instance."""
```

Comment:
Argh, I'm making work for myself here. Related to your fgpu/main.py observation @bmerry, should I update the docstring here? This does seem to lend itself more to `src` rather than `recv`; I'm happy to roll back the change here.

Reply:
I wouldn't bother changing the docstring. Things are looking good.